plugin-dev/Source/Sentry/Private/Android/Java/SentryBridgeJava.java

```diff
@@ -45,15 +45,24 @@ public void configure(SentryAndroidOptions options) {
 				options.setEnableAutoSessionTracking(enableAutoSessionTracking);
 				options.setSessionTrackingIntervalMillis(sessionTimeout);
 				options.setAttachStacktrace(enableStackTrace);
-			}
-		});
-
-		Sentry.configureScope(new ScopeCallback() {
-			@Override
-			public void run(@NonNull Scope scope) {
-				if(!gameLogPath.isEmpty()) {
-					scope.addAttachment(new Attachment(gameLogPath));
-				}
+				options.setBeforeSend(new SentryOptions.BeforeSendCallback() {
+					@Override
+					public SentryEvent execute(SentryEvent event, Hint hint) {
+						if(event.isCrashed() && event.isErrored())
```

Comment:
Doesn't that mean that if the app crashed in the run before the gamelogs from the current run won't be part of any current events?

Reply:
No, after sending the crash event on app startup all subsequent events should be treated as usual with a current session's game log attached.